docusaurus/docs/how-to-guides/panel-plugins/index.md

```diff
@@ -0,0 +1,16 @@
+---
+id: data-source-plugins
+title: Data source plugins
+description: Learn how to perform common tasks in Grafana data source plugin development.
```

Comment:
```suggestion
description: Learn how to add additional functionality and capabilities to Grafana panel plugins.
```

Reply:
deleted duplicate. will alter description in other PR